src/components/map/toolbar/EventLog.tsx

```diff
@@ -14,16 +19,86 @@ interface IEventLogProps {
 
 @observer
 export default class EventLog extends React.Component<IEventLogProps, IEventLogState> {
+    scrollRef: React.RefObject<HTMLDivElement>;
+
     constructor(props: IEventLogProps) {
         super(props);
         this.state = {
             isOpen: false,
             isAlertShown: false,
         };
+        this.scrollRef = React.createRef();
+    }
+
+    private getLogRow = (action: logActions | null, objectId: string, entity: entityNames) => {
+        let logActionClass = '';
+        switch (action) {
+        case logActions.ADD:
+            logActionClass = s.logActionAdd;
+            break;
+        case logActions.DELETE:
+            logActionClass = s.logActionDelete;
+            break;
+        case logActions.MOVE:
+            logActionClass = s.logActionMove;
+            break;
+        }
```

Comment:
There is no such events yet as DELETE / MOVE. Why not just log ADD events?

Imo, generally it is good to not code something that is not being used - or how can I test it, reviewing also a bit harder.

Also when the time comes when this is taken into use and I put a review comment on it, the coder says "it was already there"

Reply:
Okay, this is a bit two-sided. I understand your point, especially in the reviewing part. My view is that it is natural to add the different alternatives when coding a thing like this, in order to see how it would work in the end, and since it is the most effective way, since the coder has insight in the component that he is coding. I can leave the code there, but having it commented out. Which means it will need reviewing when uncommenting it.